test/e2e/network/ingress.go

```diff
@@ -634,7 +635,7 @@ var _ = common.SIGDescribe("Ingress API", func() {
 						IngressRuleValue: networkingv1.IngressRuleValue{
 							HTTP: &networkingv1.HTTPIngressRuleValue{
 								Paths: []networkingv1.HTTPIngressPath{{
-									Path:     "/",
+									Path:     "/testpath/" + fmt.Sprintf("%08x", rand.Int31()),
```

Comment:
this will be the same path too
```
		// Ingress resource create/read/update/watch verbs
		ginkgo.By("creating")
		_, err := ingClient.Create(context.TODO(), ingTemplate, metav1.CreateOptions{})
		framework.ExpectNoError(err)
		_, err = ingClient.Create(context.TODO(), ingTemplate, metav1.CreateOptions{})
		framework.ExpectNoError(err)
		createdIngress, err := ingClient.Create(context.TODO(), ingTemplate, metav1.CreateOptions{})
		framework.ExpectNoError(err)
```

Reply:
don't use the random generator, just generate 3 new ingress objects from the template with different host/path and use them in the Create calls, something like

```
                ingress1 = ingTemplate.DeepCopy()
                ingress1.Spec.Rules[0].Host = "host1.bar.com"
                ingress2 = ingTemplate.DeepCopy()
                ingress2.Spec.Rules[0].Host = "host2.bar.com"
                ingress3 = ingTemplate.DeepCopy()
                ingress3.Spec.Rules[0].Host = "host1.bar.com"
		// Ingress resource create/read/update/watch verbs
		ginkgo.By("creating")
		_, err := ingClient.Create(context.TODO(), ingress1. metav1.CreateOptions{})
		framework.ExpectNoError(err)
		_, err = ingClient.Create(context.TODO(), ingress2, metav1.CreateOptions{})
		framework.ExpectNoError(err)
		createdIngress, err := ingClient.Create(context.TODO(), ingress3, metav1.CreateOptions{})
		framework.ExpectNoError(err)
```

Reply:
yeah, this helps, will update it. Thanks !